youtube.css

```diff
@@ -0,0 +1,104 @@
+body{
```

Comment:
please try to correct linters errors,remove the extra indentation and leave single space before '{' , code without linters errors is more clean and clear and looks professional